### DISCUSSION TITLE
Difference between Debug and Release

### DISCUSSION THREAD

**Discussion by danielchalmers** [dotnet/maui#16156] [Q&A]
How can I find out every difference between running in Debug and Release modes? I'm trying to diagnose an infinite hang on the loading screen of a MAUI Blazor Hybrid app that only occurs with Release mode on Android.

Thank you for any help :)

**Comment by MauiUIui** [comment]
@danielchalmers Did you figure it out afterwards? I'm experiencing similar behaviour with RM and want to know what are the differences or how to find them out

**Reply by danielchalmers** [reply]
I was not able to get anywhere with that, but the infinite hang I was experiencing was due to a EF Core if I remember

**Reply by Ghevi** [reply]
Always delete bin and obj before building/publishing in release.

Anyway I also used EF Core and I have these settings for android:
\<PublishTrimmed>False\</PublishTrimmed>
\<RunAOTCompilation>False\</RunAOTCompilation>
\<MtouchNoSymbolStrip>False\</MtouchNoSymbolStrip>
        
And these for ios:
\<MtouchNoSymbolStrip>False\</MtouchNoSymbolStrip>
\<MtouchDebug>True\</MtouchDebug>
\<UseInterpreter>True\</UseInterpreter>

I didn't paste the applicationVersion for clarity, but i reached 42 for android and 70 for ios, just to let you know how many times i had to deploy the apps to catch all the bugs haha.
